src/main/scala/scalafix/sbt/ScalafixPlugin.scala

```diff
@@ -217,7 +217,7 @@ object ScalafixPlugin extends AutoPlugin {
       shellArgs: ShellArgs,
       config: Configuration
   ): Def.Initialize[Task[Unit]] = Def.taskDyn {
-    val dependencies = libraryDependencies.value
+    val dependencies = allDependencies.value
```

Comment:
Since libraryDependencies will be eventually added to allDependencies, I believe this change will not break sbt-scalafix user codes.